src/components/CommentInfo.tsx

```diff
@@ -0,0 +1,9 @@
+import { Comment } from '../types/Comment';
+
+export const CommentInfo: React.FC<{ comment: Comment; }> = ({ comment }) => (
```

Comment:
Let's put the type of props into the `type` variable, cause it's hard to read code like that, make sure to fix everywhere

Reply:
Sorry, my bad